site/contents/write_ups/2022/09-02-22_writeup.md

```diff
@@ -0,0 +1,69 @@
+# Data Ethics Club: [“You Social Scientists Love Mind Games”: Experimenting in the “divide” between data science and critical algorithm studies](https://journals.sagepub.com/doi/full/10.1177/2053951719833404)
+<!--Please don't edit the info panel below-->
+
+
+```{admonition} What's this? 
+This is summary of Wednesday 9th February's Data Ethics Club discussion, where we discussed the paper [“You Social Scientists Love Mind Games”: Experimenting in the “divide” between data science and critical algorithm studies](https://journals.sagepub.com/doi/full/10.1177/2053951719833404) by David Moats and Nick Seaver.
+
+The summary was written by Huw Day, who tried to synthesise everyone's contributions to this document and the discussion. "We" = "someone at Data Ethics Club". 
+Nina Di Cara and Natalie Thurlby helped with the final edit.
+
+```
+
+## Introduction
+
+This week at Data Ethics Club we discussed the paper “You Social Scientists Love Mind Games”: Experimenting in the “divide” between data science and critical algorithm studies by David Moats and Nick Seaver. The paper examines the divide between and intersection between data scientists and social scientists using methodology more familiar to anthropologists than those with a STEM background:
+
+"we attempted to stage an encounter in which practicing data scientists were asked to analyze a corpus of critical social science literature about their work, using tools of textual analysis such as co-word and topic modelling. The idea was to provoke discussion both about the content of these texts and the possible limits of such analyses."
+
+We discussed the authors' approach, the divides between social and data science and what we can do to bridge this gap. We see some reoccuring themes when data scientists try to bring their tools to bear on social science problems and vice versa: https://xkcd.com/1831/ 
+
+## What do you think about the authors’ approach (framing of the experiment, the first vs the second attempt, tools suggested, their reflections, Nick’s email)?
+
+We thought the experiment was interesting, getting data scientists to be reflective in a way that is not typical of their skillset. Some of us felt the experiment lacked specificity about what data scientists actually do (using quantitative methods to answer specific questions). Perhaps it could have been more successful by asking them to read the papers and discuss them, as opposed to getting them to use these tools to analyse them with quantative methods. It didn't seem like there was a way for the participants to succeed with their analysis. 
```

Comment:
```suggestion
We thought the experiment was interesting, getting data scientists to be reflective in a way that is not typical of their skillset, though some of us felt the experiment lacked specificity about what data scientists actually do (using quantitative methods to answer specific questions). 
Perhaps it could have been more successful by asking the data scientists to read the papers and discuss them, as opposed to getting them to analyse them with quantitative methods. It didn't seem like there was a way for the participants to succeed with their analysis. 
```